src/unstract/adapters/vectordb/postgres/src/postgres.py

```diff
@@ -58,7 +59,8 @@ def get_vector_db_instance(self) -> BasePydanticVectorStore:
 
     def _get_vector_db_instance(self) -> BasePydanticVectorStore:
         try:
-
+            encoded_password = urllib.parse.quote_plus(
+                str(self._config.get(Constants.PASSWORD))
```

Comment:
Looks like this change is missed 

```suggestion
            encoded_password = quote_plus(
                str(self._config.get(Constants.PASSWORD)
            )
```